src/vs/workbench/contrib/terminal/common/scripts/shellIntegration.ps1

```diff
@@ -169,6 +169,14 @@ elseif ((Test-Path variable:global:GitPromptSettings) -and $Global:GitPromptSett
 	[Console]::Write("$([char]0x1b)]633;P;PromptType=posh-git`a")
 }
 
+
+if (-not (Get-Module -Name PSReadLine) -and $env:VSCODE_A11Y_MODE) {
+	$scriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path
+	$specialPsrlPath = Join-Path $scriptRoot '\psreadline\'
```

Comment:
Guessing you don't need the `\`? Looks wrong seeing that since this is meant to work across platforms.